R/roxygen.R

```diff
@@ -0,0 +1,124 @@
+#' @importFrom roxygen2 roxy_tag_parse
+#' @importFrom roxygen2 roxy_tag_rd
```

Comment:
I think you could drop these and switch from `@export`ing the method to `@S3method` registering it so you don't need to import roxygen2.

Reply:
Sorry, I meant `@exportS3Method`, https://roxygen2.r-lib.org/articles/namespace.html#s3